File: recovered-source/jadx/sources/me/ag2s/epublib/util/commons/io/XmlStreamReaderException.java
package me.ag2s.epublib.util.commons.io;

import java.io.IOException;

/* JADX INFO: loaded from: reader-pro-classes-3.2.14.jar:me/ag2s/epublib/util/commons/io/XmlStreamReaderException.class */
public class XmlStreamReaderException extends IOException {
    private static final long serialVersionUID = 1;
    private final String bomEncoding;
    private final String xmlGuessEncoding;
    private final String xmlEncoding;
    private final String contentTypeMime;
    private final String contentTypeEncoding;

    public XmlStreamReaderException(final String msg, final String bomEnc, final String xmlGuessEnc, final String xmlEnc) {
        this(msg, null, null, bomEnc, xmlGuessEnc, xmlEnc);
    }

    public XmlStreamReaderException(final String msg, final String ctMime, final String ctEnc, final String bomEnc, final String xmlGuessEnc, final String xmlEnc) {
        super(msg);
        this.contentTypeMime = ctMime;
        this.contentTypeEncoding = ctEnc;
        this.bomEncoding = bomEnc;
        this.xmlGuessEncoding = xmlGuessEnc;
        this.xmlEncoding = xmlEnc;
    }

    public String getBomEncoding() {
        return this.bomEncoding;
    }

    public String getXmlGuessEncoding() {
        return this.xmlGuessEncoding;
    }

    public String getXmlEncoding() {
        return this.xmlEncoding;
    }

    public String getContentTypeMime() {
        return this.contentTypeMime;
    }

    public String getContentTypeEncoding() {
        return this.contentTypeEncoding;
    }
}

File: src/main/kotlin/me/ag2s/epublib/util/commons/io/XmlStreamReaderException.kt
package me.ag2s.epublib.util.commons.io

import java.io.IOException

class XmlStreamReaderException : IOException {
    private val bomEncoding: String?
    private val xmlGuessEncoding: String?
    private val xmlEncoding: String?
    private val contentTypeMime: String?
    private val contentTypeEncoding: String?
    constructor(message: String, bomEncoding: String?, xmlGuessEncoding: String?, xmlEncoding: String?) : this(message, null, null, bomEncoding, xmlGuessEncoding, xmlEncoding)
    constructor(message: String, contentTypeMime: String?, contentTypeEncoding: String?, bomEncoding: String?, xmlGuessEncoding: String?, xmlEncoding: String?) : super(message) { this.contentTypeMime = contentTypeMime; this.contentTypeEncoding = contentTypeEncoding; this.bomEncoding = bomEncoding; this.xmlGuessEncoding = xmlGuessEncoding; this.xmlEncoding = xmlEncoding }
    fun getBomEncoding(): String? = bomEncoding
    fun getXmlGuessEncoding(): String? = xmlGuessEncoding
    fun getXmlEncoding(): String? = xmlEncoding
    fun getContentTypeMime(): String? = contentTypeMime
    fun getContentTypeEncoding(): String? = contentTypeEncoding
    companion object { private const val serialVersionUID = 1L }
}
